### PR TITLE
Mayhem Heroes Integration

### PR DESCRIPTION
**Pull request by capuanob**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Build Stage
```
Comment (xansec) [inline Dockerfile:1]: Note that corpus -> testsuite. No need to run git clone here.

File: Mayhemfile [hunk 2]
```diff
@@ -0,0 +1,10 @@
+image: ghcr.io/capuanob/sdl-sound-fuzz:latest
```
Comment (xansec) [inline Mayhemfile]: Please omit the image, and change tests -> testsuite.

File: fuzz/fuzz.c [hunk 3]
```diff
@@ -0,0 +1,29 @@
+#include <stdlib.h>
```
Comment (xansec) [inline fuzz/fuzz.c]: Can we rename this to make it clear what's being fuzzed?

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

File: Dockerfile (hunk 1 above)
Comment (xansec) [inline Dockerfile:1]: Also, you should not use git clone. Please use `ADD` instead.

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

**Comment by capuanob** [comment]
@xansec Made these changes, please let me know if anything else is needed!

**Review by xansec — APPROVED** [review]
LGTM!

**Pull request merged**